packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -595,4 +596,28 @@ export class Interpreter {
 		}
 		return result;
 	}
+
+	/**
+	 * This functions checks if the requested round is avaiable to access. If it is
+	 * not throws an Error
+	 * @param round: round number
+	 * @returns void
+	 */
+	assertRoundIsAvailable(round: number): void {
+		let firstAvail = this.runtime.ctx.tx.lv - MaxTxnLife - 1;
+		if (firstAvail > this.runtime.ctx.tx.lv || firstAvail === 0) {
+			// early in chain's life
+			firstAvail = 1;
+		}
+		let lastAvail = this.runtime.ctx.tx.fv === undefined ? 0 : this.runtime.ctx.tx.fv - 1;
```

Comment:
```suggestion
		let lastAvail = this.runtime.ctx.tx.fv ?  this.runtime.ctx.tx.fv - 1 :0;
```

Reply:
and this value can be undefined. This is type from algosdk